src/Abstractions/NexusMods.Abstractions.Loadouts/Mods/Operations.cs

```diff
@@ -0,0 +1,52 @@
+using NexusMods.Abstractions.Loadouts.Ids;
+using NexusMods.MnemonicDB.Abstractions;
+using NexusMods.MnemonicDB.Abstractions.TxFunctions;
+
+using File = NexusMods.Abstractions.Loadouts.Files.File;
+
+namespace NexusMods.Abstractions.Loadouts.Mods;
+
+public static partial class Mod
+{
+
+    public partial class Model
+    {
+        /// <summary>
+        /// Toggle the enabled status of the mod.
+        /// </summary>
+        public async Task ToggleEnabled()
+        {
+            using var tx = Db.Connection.BeginTransaction();
+            var old = Db.Get(ModId);
+            tx.Add(ModId, static (tx, db, id) =>
+                {
+                    var old = db.Get(id);
+                    tx.Add(id.Value, Mod.Enabled, !old.Enabled);
+                }
+            );
+            old.Loadout.Revise(tx);
+            
+            old.Revise(tx);
+            await tx.Commit();
+        }
+        
+        /// <summary>
+        /// Toggle the enabled status of the mod.
+        /// </summary>
```

Comment:
That's just wrong.

Reply:
Oh right, copied comment and I was looking at the wrong method